CHANGES.rst

```diff
@@ -1,6 +1,94 @@
 Release Notes
 =============
 
+2.1.25
+------
+
+This release brings support for a ``--venv`` execution mode to complement ``--unzip`` and standard
+unadorned PEX zip file execution modes. The ``--venv`` execution mode will first install the PEX
+file into a virtual environment under ``${PEX_ROOT}/venvs`` and then re-execute itself from there.
+This mode of execution allows you to ship your PEXed application as a single zipfile that
+automatically installs itself in a venv and runs from there to eliminate all PEX startup overhead
+on subsequent runs and work like a "normal" application.
+
+There is also support for a new resolution mode when building PEX files that allows you to use the
+results of a previous resolve by specifying it as a ``-pex-repository`` to resolve from. If you have
+many applications sharing a requirements.txt / constraints.txt this can drastically speed up
```

Comment:
Grammar nit:

```suggestion
many applications sharing a requirements.txt / constraints.txt, this can drastically speed up
```

Reply:
I'll save a tree. Any grammar that you can actually parse clearly works already, No-one is confused here fwict.